src/darwin/Framework/CHIP/templates/availability.yaml

```diff
@@ -10084,9 +10080,245 @@
+      enum values:
+          MicrowaveOvenMode:
+              ModeTag:
+                  # generic mode tags
+                  - Auto
+                  - Quick
+                  - Quiet
+                  - LowNoise
+                  - LowEnergy
+                  - Vacation
+                  - Min
+                  - Max
+                  - Night
+                  - Day
+                  # MicrowaveOvenMode-specific tags
+                  - Normal
+                  - Defrost
+          OvenMode:
+              ModeTag:
+                  # generic mode tags
+                  - Auto
+                  - Quick
+                  - Quiet
+                  - LowNoise
+                  - LowEnergy
+                  - Vacation
+                  - Min
+                  - Max
+                  - Night
+                  - Day
+                  # OvenMode-specific tags
+                  - Bake
+                  - Convection
+                  - Grill
+                  - Roast
+                  - Clean
+                  - ConvectionBake
+                  - ConvectionRoast
+                  - Warming
+                  - Proofing
+                  - Steam
+          OvenCavityOperationalState:
+              ErrorStateEnum:
+                  - NoError
+                  - UnableToStartOrResume
+                  - UnableToCompleteOperation
+                  - CommandInvalidInState
+              OperationalStateEnum:
+                  - Stopped
+                  - Running
+                  - Paused
+                  - Error
+      bitmaps:
+          MicrowaveOvenControl:
+              - Feature
+      bitmap values:
+          MicrowaveOvenControl:
+              Feature:
+                  - PowerAsNumber
+                  # PowerInWatts is provisional because WATTS is provisional in Matter 1.3
+                  # WATTS is provisional in 1.3 because no one has submitted a DUT for it
+                  - PowerNumberLimits
   provisional:
       device types:
           - BatteryStorage
           - HeatPump
           - SolarPower
           - WaterHeater
+      attributes:
+          MicrowaveOvenControl:
+              - SupportedWatts
+              - SelectedWattIndex
+      command fields:
+          MicrowaveOvenControl:
+              SetCookingParameters:
+                  - wattSettingIndex
+      bitmap values:
+          MicrowaveOvenControl:
+              Feature:
+                  - PowerInWatts
+  removed:
+      attributes:
+          OvenMode:
+              - StartUpMode
+              - OnMode
+      commands:
+          OvenCavityOperationalState:
+              - Pause
+              - Resume
+      bitmaps:
+          # MicrowaveOvenMode and OvenMode Feature bitmaps removed because
+          # an NS_ENUM must have at least one value and without ONOFF they have none
+          MicrowaveOvenMode:
+              - Feature
+          OvenMode:
+              - Feature
+      bitmap values:
```

Comment:
Why do you need these?  if the bitmap is removed, then the bits inside it are removed automatically....  I guess the bitmap values removals got added first to implement the X from the spec, and then that failed to compile and the bitmap removals above were added?

In any case, either just remove the bitmaps or clearly document why the redundant bits are here (e.g. so that if the bitmaps ever get un-removed due to growing new bits we will not get incorrect bits in them).